pkg/cli/admin/release/info.go

```diff
@@ -61,7 +61,7 @@ func NewInfoOptions(streams genericclioptions.IOStreams) *InfoOptions {
 func NewInfo(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
 	o := NewInfoOptions(streams)
 	cmd := &cobra.Command{
-		Use:   "info IMAGE [--changes-from=IMAGE] [--verify|--commits|--pullspecs]",
+		Use:   "info IMAGE [--changes-from=IMAGE] [--verify|--commits|--pullspecs|--arch=ARCH]",
```

Comment:
This seems to overlap with `--filter-by-os`?  Do we need a new option?

Reply:
@wking agree, removed --arch logice, using --filter-by-os instead.